openrr-teleop/src/move_base.rs

```diff
@@ -116,4 +117,95 @@ mod tests {
         assert!(!node.is_enabled);
         assert!(!node.is_turbo);
     }
+
+    #[test]
+    fn test_move_node_get() {
+        let mode = String::from("tested");
+        let base = DummyMoveBase::new();
+        let node = MoveBaseNode::new(mode.clone(), base);
+
+        let base_mode = node.mode();
+        assert_eq!(base_mode, mode);
+        let base_sub_mode = node.submode();
+        assert_eq!(base_sub_mode, &String::from(""));
+    }
+
+    #[tokio::test]
+    async fn test_move_node_proc() {
+        let mode = String::from("tested");
+        let x = 1.2_f64;
+        let y = 3.5;
+        let theta = 1.8;
```

Comment:
Why added _f64 only for `x`?
Can't compile without it?

Reply:
And you can use const if the values are not changed.

```rust
const x: f64 X = 1.2;
...
```

Reply:
And it's better to add some information about what x means.
(reference, target, initial, or something)
I mean `const SOME_INFORMATION_X: f64 = 1.2;` is better.

Reply:
Other valiables are forgot to add it.

But, no explicit type is judged `float` probably.
I think compiler miss to read true sentences in test function, it is not test.
Using implicit delaration is good for reading when you use it in practical scene.
I use usually post-modification, now I heard your opnion and I'm going to use pre-modification if I notice it.